optionlab/plot.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import division
```

Comment:
My dream for this function is to use plotly. I will put it in the #wishlist @ discord.

Reply:
Plotly is a much better library, I am pretty familiar with it, less so in Python but can get by

Reply:
It would be nice. My son showed me some plotly's plots, definitely it is another level compared to simple (and ugly) matplotlib plots... But I had not time to learn how to use plotly.